apps/reflect/src/forms/TwitchBotForm.tsx

```diff
@@ -158,7 +170,14 @@ const TwitchBotForm = () => {
               <Select
                 onChange={field => {
                   onChange(field.target.value);
-                  setButtonEnabled(true);
+                  if (
+                    configData[1]?.key === "mods_manage_cmds_enabled" &&
+                    configData[1].value === field.target.value
+                  ) {
+                    setButtonEnabled(false);
```

Comment:
What if we use something like this here?
```ts
setButtonEnabled(configData[1]?.key !== "mods_manage_cmds_enabled" && configData[1].value !== field.target.value);
```

Would it make the code harder to read?

Reply:
Hello, thank you very much for this this code looks shorter